src/en/components/checkbox/base.md

```diff
@@ -1,14 +1,42 @@
 ---
+title: Checkbox
 layout: "layouts/base.njk"
 github: https://github.com/cds-snc/gcds-components/tree/main/src/components/gcds-checkbox
 figma: https://www.figma.com/community/file/1128687821123298228
 permalink: false
-tags: ['checkboxEN', 'header']
+tags: ["checkboxEN", "header"]
 ---
 
-# checkbox
+# {{ title }}
 
-This element will have the content below it
+`<gcds-checkbox>`
```

Comment:
Does it look better if you use the syntax highliter aka:
```html
<gcds-checkbox>
```

Reply:
I can try this

Reply:
To be honest, I think both looks a bit out of place :D :D :D 

The dark bg is very coding focused, and the gray bg is gigantic and not code looking enough.

Reply:
I'll just do exactly as in Figma and you can change it later

Reply:
Sounds good, thanks for trying different options! Part of me wants to remove this now :D But we'll see what it looks like when we do a review next week and how everyone is feeling about it.

Reply:
I think the "user select" stuff was neat, it was super easy to click and copy. I agree that at first, the big code tag seemed weird, even in heading font. My only concern now is a11y and how it will get read by screen readers.